article/helpers.py

```diff
@@ -40,6 +43,19 @@ def persist_article(self, article_uuid):
     def retrieve_articles(self):
         return self.request.session.get(self.SESSION_KEY, [])
 
+    def article_read_count(self, category):
+        if category not in structure.CATEGORIES_LOOKUP:
+            raise Exception('Incorrect category')
```

Comment:
A similar error will be raised by `CATEGORIES_LOOKUP[category]` - I would remove this unless we want to catch this specific error message for some reason and expect it to ever happen.